File: back/src/main/java/csh/back/domain/member/dto/web/LoginResult.java
package csh.back.domain.member.dto.web;

import csh.back.domain.member.dto.response.LoginResponseDto;

// 로그인 결과 (사용자 정보 + 토큰을 컨트롤러에 전달하기 위한 내부 타입)
public record LoginResult(
        LoginResponseDto userInfo,
        String accessToken,
        String refreshToken
) {}

File: back/src/main/kotlin/csh/back/domain/member/dto/web/LoginResult.kt
package csh.back.domain.member.dto.web

import csh.back.domain.member.dto.response.LoginResponseDto

// Service → Controller 내부 전달용 (JSON 직렬화 대상 아님)
data class LoginResult(
    val userInfo: LoginResponseDto,
    val accessToken: String,
    val refreshToken: String,
)